.circleci/regenerate.py

```diff
@@ -28,6 +28,10 @@
 
 def build_workflows(prefix="", filter_branch=None, upload=False, indentation=6, windows_latest_only=False):
     w = []
+
```

Comment:
nit: We can probably just remove this function and template entirely if there are no build_workflows.

Reply:
Agree! Once we confirm that it works will remove this completely